python-backend/tests/mines/resources/test_mine_subscription_resource.py

```diff
@@ -0,0 +1,37 @@
+import json
+from tests.constants import (TEST_MINE_NAME, TEST_MINE_GUID)
```

Comment:
test suite needs to be updated to use factories.

Reply:
done